src/static/js/chapter.js

```diff
@@ -0,0 +1,134 @@
+//Data Save can be set to on, so let's check it
+function dataSaverEnabled() {
+  let dataSaver = false;

```

Comment:
FWIW let and cost have the same syntax issues as async/await. I think we should be entirely ES6 or not at all.

Reply:
OK fixed that nite... but also came up with the solution to my problem!!